### PR TITLE
[WIP] Add relations!

### PR DESCRIPTION
**Pull request by nathunsmitty**
Takes the Show, User, and SocialLink models created by @tanzeelak, @kianadk, and @xualexander respectively and relates them together. Yay relational databases!

I changed a couple properties and probably still have some `any`s in places so please call me out on those. I'm also having trouble with figuring out how to store show times, but I also should do some math homework rn so please comment below this asking me what my struggle is. This will also let me know you actually read my PRs! :)

Also merged all the create data scripts because its tough to relate things when they're spread across scripts. I don't think I actually added ShowLink into this new script as well so please call me out on that too.

### CODE REVIEW

File: src/models/Show.ts [hunk 1]
```diff
@@ -13,21 +13,33 @@ export interface ShowInstance extends Sequelize.Instance<ShowAttributes> {
...
+  genre: string;
+  imageURL: string;
+
+  day: Day;
+  startTime: string;
+  duration: number;
+}
 
-Show.belongsToMany(User, { through: 'UserShow' });
+const Show: Sequelize.Model<any, any> = sequelize.define('Show', {
```
Comment (xualexander) [inline src/models/Show.ts]: `any`s

File: src/models/User.ts [hunk 2]
```diff
@@ -5,22 +5,21 @@ export interface UserAttributes {}
...
-    djName: Sequelize.STRING,
-    email: Sequelize.STRING,
-    fullName: Sequelize.STRING,
-    password: Sequelize.STRING,
-    phone: Sequelize.STRING,
-    picture: Sequelize.STRING,
-    shows: Sequelize.ARRAY(Sequelize.TEXT),
-  }
-);
+const User: Sequelize.Model<any, any> = sequelize.define('User', {
```
Comment (xualexander) [inline src/models/User.ts]: `any`s

File: src/schema/show.graphql [hunk 3]
```diff
@@ -19,8 +19,8 @@ scalar Date
 type Show {
   id: ID!
   title: String!
-  startTime: Date!
-  endTime: Date!
+  nextShowDate: Date!
+  duration: Int!
   djs: [DJ]!
```
Comment (xualexander) [inline src/schema/show.graphql]: Do we need an array of social media as well?

File: src/models/Show.ts [hunk 4]
```diff
@@ -13,21 +16,33 @@ export interface ShowInstance extends Sequelize.Instance<ShowAttributes> {
...
-    description: Sequelize.STRING,
-    genre: Sequelize.STRING,
-    image: Sequelize.STRING,
-    startTime: Sequelize.TIME,
-    endTime: Sequelize.TIME,
-  }
-);
-
-Show.belongsToMany(User, { through: 'UserShow' });
+const Show: Sequelize.Model<any, any> = sequelize.define('Show', {
```
Comment (xualexander) [inline src/models/Show.ts]: `any`s

File: src/schema/show.graphql (hunk 3 above)
Comment (nathunsmitty) [reply src/schema/show.graphql]: We do, but I'm gonna leave that for another PR. If you're interested in working on graphql, I'd recommend you take it!

**Pull request merged**